src/metabase/models/setting.clj

```diff
@@ -744,7 +749,8 @@
                  :cache?         true
                  :database-local :never
                  :user-local     :never
-                 :deprecated     nil}
+                 :deprecated     nil
+                 :enabled?       nil}
```

Comment:
One option could be to make the default value for `enabled?` `(constantly true)` so that you don't need to worry about the `(nil? enabled)` check in the getter

Reply:
I think I'm going to leave it as is. My thinking is that `enabled?` is an optional property and the default is of course settings just have values. I like that its an optional policy and we aren't adding lots of functions to most of them. If we live to regreat this we can easily add the `(constantly true)` and drop the nil checks.